docs/csharp/language-reference/operators/conditional-operator.md

```diff
@@ -80,14 +80,14 @@ The conditional operator cannot be overloaded.
 
 ## C# language specification
 
-For more information, see the [Conditional operator](~/_csharplang/spec/expressions.md#conditional-operator) section of the [C# language specification](../language-specification/index.md).
+For more information, see the [Conditional operator](~/_csharplang/spec/expressions.md#conditional-operator) section of the [C# language specification](~/_csharplang/spec/introduction.md).
 
 ## See also
 
 - [C# Reference](../index.md)
 - [C# Programming Guide](../../programming-guide/index.md)
```

Comment:
@BillWagner is that link to the programming guide still useful? Are those used (and if used, the final result is satisfying?)? Given future restructures of the C# doc set, would it be fine to remove that link from the **See also** sections already now?

Reply:
I agree that we can remove these.

The data shows they aren't clicked on much.

Reply:
@BillWagner thanks for the check. I'll submit a separate PR with those sections clean-up, then.